verification/src/http_server/handlers/verification/solidity/version_list.rs

```diff
@@ -0,0 +1,21 @@
+use super::types::VersionsResponse;
+use crate::{
+    compiler::{download_cache::DownloadCache, fetcher::Fetcher},
+    solidity::github_fetcher::GithubFetcher,
+};
+
+use actix_web::{
+    web::{self, Json},
+    Error,
+};
+
+pub async fn get_version_list(
+    cache: web::Data<DownloadCache<GithubFetcher>>,
+) -> Result<Json<VersionsResponse>, Error> {
+    let mut versions = cache.fetcher().all_versions();
+    // sort in descending order
+    versions.sort_by(|x, y| y.cmp(x));
```

Comment:
actually, I think that it would be better, if `all_versions` returned already sorted vector. There is no obvious way to me to make it clear in the trait (except commenting), but the idea is that this list is changed very rarely (once a day?) and we can sort it once a day and reuse the sorted vector after that 
and we could implement it in the download cache, for example?
but okay, this is some ammount of work that we probably don't need right now, I'll just make an issue about that

Reply:
Hm, maybe just change HashMap to BTreeMap? Then, we will need just to reverse list in handler function